test/TestScheduleTx.ts

```diff
@@ -44,6 +44,10 @@ export const getHelperMethods = (web3 : any) => {
   return { waitUntilBlock };
 };
 
+export const SCHEDULED_TX_PARAMS = {
+  callValue: new BigNumber(Math.pow(10, 18))
```

Comment:
Might be better to use `web3.eth.toWei(10, 'ether')` which generates a BigNumber.

Reply:
Actually `web3.toWei(10, 'ether')` returns a string.

Reply:
It returns a string if you provide it with a string. Gives a BigNumber if you provide it with a BigNumber. https://github.com/ethereum/wiki/wiki/JavaScript-API#returns-16